### ISSUE TITLE
Question about measuring the data pruning metrics

### ISSUE THREAD

**Issue by Jasonlee1995** [rgeirhos/dataset-pruning-metrics#3]
Hi.

I recently read your paper and have a question about the training settings used to measure the metrics of single data point.

<img width="600" alt="image" src="https://github.com/rgeirhos/dataset-pruning-metrics/assets/49643709/80137b7a-29b3-4f52-af71-8e61f57a4306">

[ImageNet training recipe link](https://github.com/facebookresearch/vissl/blob/v0.1.6/configs/config/pretrain/supervised/supervised_8gpu_resnet.yaml)

In `Appendix B`, I find the ImageNet training recipe.

But I'm curious if the same training recipe is used to measure the data pruning metrics of a single data point?
(e.g. to measure EL2N score for the ImageNet data, did you use the same training recipe as above?)

**Comment by rgeirhos** [comment]
Hi @Jasonlee1995 thanks for your interest. I can confirm that we used the exact same training recipe irrespective of the choice of dataset pruning metric for comparability.
